inc/thorvg.h

```diff
@@ -170,49 +179,47 @@ struct Matrix
 /**
  * @class Paint
  *
- * @brief A abstract class for managing graphical elements.
- *
- * Basically a graphical elements in TVG is composed as an object in a scene (Canvas).
- * Paint represents a graphical object, suggest common behaviors such as duplication, transformation and composition, etc.
- * TVG recommends user to regard a paint as a pack of volatile commands, they can prepare a paint then request Canvas to run it.
```

Comment:
Do you need to remove the last sentence? It's a tip for tvg concept.

Reply:
@mgrudzinska volatile means commands are just one single input that turns to invalid soon. So you suggestion can't satisfy with the meaning.

Reply:
restored you version:
"TVG recommends user to regard a paint as a set of volatile commands. They can prepare a Paint and then request a Canvas to run them."